src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -2006,7 +2008,7 @@ public override DeclarationKind GetDeclarationKind(SyntaxNode declaration)
                         return DeclarationKind.Attribute;
                     }
                     break;
-
+                // Add InitAccessor once https://github.com/dotnet/roslyn/issues/48136 is resolved
```

Comment:
I don't think this comment is adding any value.

Reply:
Removed it.